routers/web/repo/issue.go

```diff
@@ -1248,9 +1256,9 @@ func ViewIssue(ctx *context.Context) {
 	}
 
 	var (
-		tag          models.CommentTag
+		tag          models.RoleDescriptor
```

Comment:
Should this `tag` be named to `roleDescriptor` as above?

Reply:
`roleDescriptor` is not possible as that's already the name of the function to get the RoleDescriptor, I think it can be either `role` or `roleLabel`.